lib/views/pages/login_signup/set_url_page.dart

```diff
@@ -68,7 +67,9 @@ class _UrlPageState extends State<UrlPage>
       LogHelper().log(LogLevel.ERROR, widget.toStringShort(), "checkAndSetUrl",
           "Incorrect Oraganization",
           exception: e as Exception);
-      _exceptionToast('Incorrect Organization Entered');
+      Fluttertoast.showToast(
+          msg: 'Incorrect Organization Entered',
```

Comment:
error messages need to be abstracted out to internationalization file. multiple languages are supported.

Reply:
Hi @dellyjm, as per my knowledge multilingual support is not added yet, it's a part of the GSoC project.
Please see this link : [GSoC ideas](https://palisadoesfoundation.github.io/talawa-docs/docs/internships/gsoc/gsoc-ideas/#multilingual-support)

Reply:
> error messages need to be abstracted out to internationalization file. multiple languages are supported.

@dellyjm  We'll open issues on internationalization next week. Like @utkarshshendge said, some GSoC applications may include this feature and it'll be best to wait and see whether any of these are accepted.